R/data.table.R

```diff
@@ -12,9 +12,16 @@ methods::setPackageName("data.table",.global)
 #   (2) export() in NAMESPACE
 #   (3) add to vignettes/datatable-importing.Rmd#globals section
 .SD = .N = .I = .GRP = .NGRP = .BY = .EACHI = NULL
+J = function(...) {
+  stopf("The function 'J' should not be called here. 'J' is intended for use within the 'j' expression of data.table operations")
+}
+
+. = function(...) {
+  stopf("The symbol '.' should not be used here. '.' is intended for use within data.table operations.")
```

Comment:
'.' is intended for use within data.table operations -> '.' can be used as an alias for 'list' within the square brackets of a data.table, DT[.]

Reply:
done 👍